docs/pages/reference/cli.mdx

```diff
@@ -6,8 +6,8 @@ h1: Command Line (CLI) Reference
 
 Teleport is made up of three CLI tools.
 
-- [teleport](#teleport): The daemon that runs the Auth Service, Proxy Service, and other Teleport services.
-- [tsh](#tsh): A tool that lets end users interact with Teleport.
+- [teleport](#teleport): A CLI tool that supports the Teleport access plane.
+- [tsh](#tsh): A CLI tool that allows end users to interact with Teleport sessions and list cluster information.
 - [tctl](#tctl): An administrative tool that can configure the Teleport Auth Service.
```

Comment:
If the other two are described as a CLI tool, `tctl` certainly should be as well.